src/v/utils/fragmented_vector.h

```diff
@@ -222,6 +222,57 @@ class fragmented_vector {
         _capacity = 0;
     }
 
+    /**
+     * Return the number of elements the vector has allocated space for.
+     */
+    size_t capacity() const { return _capacity; }
+
+    /**
+     * Resizes the vector to contain \p count elements. If the current size is
+     * less than \p count then default-initialized elements are added. If the
+     * current size is larger then elements are removed.
+     *
+     * The expected use case for this is to allocate a large vector in a fiber
+     * using a series of smaller resize() invocations allowing for cooperative
+     * yield calls to be inserted to avoid reactor stalls. The optimal strategy
```

Comment:
I don't think there are any magic numbers here. Since fragmented vector isn't futurized, it be the same concern as resizing a std::vector or avoiding calling fragmented_vector::copy on a large fragmented vector.